src/main/java/no/nav/veilarbaktivitet/aktiviteter_til_kafka/AktiviteterTilKafkaService.java

```diff
@@ -1,4 +1,4 @@
-package no.nav.veilarbaktivitet.aktiviteterTilKafka;
+package no.nav.veilarbaktivitet.aktiviteter_til_kafka;
```

Comment:
Hva er grunnen til at vi renamer til å bruke underscore her?

Reply:
Sonar sier det er standaren for pakker i java.
Prøver egentlig og finne ut hva som er best.

Reply:
Er ingen andre pakker her som har "flere ord" i navnet så syns det er greit nok selv om det er litt uvant.